youtube_dl/downloader/__init__.py

```diff
@@ -30,6 +31,9 @@ def get_suitable_downloader(info_dict, params={}):
     protocol = determine_protocol(info_dict)
     info_dict['protocol'] = protocol
 
+    if (info_dict.get('start_time') or info_dict.get('end_time')) and FFmpegFD.supports(info_dict):
```

Comment:
This is also questionable whether we want this behavior by default at all, since it may possibly cause out of sync media in case of DASH when extracted pieces have too approximate boundaries. Have you tried downloading DASH video+audio with this approach?